.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,52 +1,67 @@
-## Pull Request (PR) Checklist
+## Pull request (PR) checklist
+
 Please check if your pull request fulfills the following requirements:
+
+<!-- 💡 Tip: Tick checkboxes like this: [✅]. -->
```

Comment:
Checkboxes are checked using `[x]` or from the UI after the PR is created

- [x] Demo